post_installation_configuration/vsphere-failure-domain-configuration.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+:context: post-install-vsphere-failure-domain-configuration
+[id="post-install-vsphere-failure-domain-configuration"]
+=  Multiple failure domain configuration for a cluster on vSphere
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As an administrator, you can specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance. This means that you can distribute key control planes and workload elements among varied hardware resources for a datacenter.
+
+A failure domain describes a unique topology. A failure domain configuration lists parameters that create a topology. The following list states some of these parameters:
+
+* `computeCluster`
+
+* `datacenter`
+
+* `datastore`
+
+* `port-group`
+
+* `resourcePool`
+
+
+After you define failure domains for your {product-title} cluster, you can create or migrate nodes to another failure domain.
+
+[IMPORTANT]
+====
+If you want to migrate pre-existing {product-title} cluster compute nodes to a failure domain, you must define a new compute machine set for the compute node. This new machine set can scale up a compute note according to the topology of the failure domain, and scale down the pre-existing compute note. 
```

Comment:
'scale up a compute note' --> scale up a compute node

Reply:
Hi @sgaoshang . I fixed the typos. Thanks for spotting these.

Please let me know if you QE approve the content?